velox/docs/develop/cache.rst

```diff
@@ -0,0 +1,152 @@
+===========================
+AsyncDataCache (File Cache)
+===========================
+
+Background
+----------
+Velox provides transparent file cache (AsyncDataCache) to accelerate table scan through the hot data reuse and prefetch. The file cache is integrated with the memory system to achieve dynamic memory sharing between file cache and query memory. When a query fails to allocate memory, we retry the allocation by shrinking the file cache. Therefore, the file cache size is automatically adjusted in response to the query memory usage change. See `Memory Management - Velox Documentation <https://facebookincubator.github.io/velox/develop/memory.html>`_  for more information about Velox's file cache.
```

Comment:
1. missing "a" -> Velox provides **a** transparent file cache

Better structure and grammar of first sentence perhaps:

Velox provides a transparent file cache (AsyncDataCache) to accelerate table scans operators through hot data reuse and prefetch algorithms.

2. missing "the"
"between **the** file cache"